scripts/3_deadlines_calendar.py

```diff
@@ -1,24 +1,25 @@
 import argparse
+from datetime import datetime
 from typing import Final
 
-from utils import validate_date
-
 TEMPLATE: Final[
     str
-] = """
-
-## Deadlines & upcoming schedule
-`{prs_merge}`: @all have all your PRs intended to be tested merged by the end of the day
-`{staging_release}`: STAGING release
-`{review_meeting}`: REVIEW MEETING
-`{review_meeting}`: by the end of the day created draft of the release notes
-`{start_app_team}`: -> {end_app_team}: App Team has exclusive access to the AWS Staging environment
-`{prod_release}`: PROD release
+] = """## Deadlines & upcoming schedule
+{sorted_events}
```

Comment:
I find this version way less readable. I preferred the declarative template approach.